src/hsmm_mvpy/utils.py

```diff
@@ -1344,9 +1375,8 @@ def event_times(data, times, channel, stage):
     return brp_data    
     
 def condition_selection(hmp_data, epoch_data, condition_string, variable='event'):
-    unstacked = hmp_data.unstack().where(epoch_data[variable].str.contains(condition_string),drop=True)
-    stacked = stack_data(unstacked)
-    return stacked
+    return hmp_data.where(hmp_data[variable] == condition_string, drop=True)
```

Comment:
Does this have a link with the recent weirdness in condition selection? Also contains is useful (e.g. Tutorial 0, raw data section), so we should rather have a parameter, e.g. method' that by default takes 'equal' but can also be adjusted to 'contains'

Reply:
No because for the condition code you actually do not split the data in conditions. I made this change because the original version crashed my code, and this seemed much simpler and do the same job. I'll add `contains` as an option.